aerogear-windows-otp/Totp.cs

```diff
@@ -6,18 +6,37 @@ namespace AeroGear.OTP
 {
     public class Totp
     {
+        public enum Digits
```

Comment:
Is this enum really necessary? DIGITS_POWER has more options then exposed in this enum, why not 'just' have an int for number of digits?